providers-sdk/v1/upstream/gql/policy_gql.go

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package gql
+
+import (
+	"context"
+	mondoogql "go.mondoo.com/mondoo-go"
+)
+
+type ContentSearchResponse struct {
+	TotalCount int          `json:"totalCount"`
+	Edges      []PolicyEdge `json:"edges"`
+	PageInfo   PageInfo     `json:"pageInfo"`
+}
+
+type PolicyEdge struct {
+	Cursor string     `json:"cursor"`
+	Node   PolicyNode `json:"node"`
+}
+
+type PolicyNode struct {
+	// The Policy struct is embedded to match the inline fragment ... on Policy.
+	// We assume that all these fields are part of the Policy type.
+	*Policy `graphql:"... on Policy"`
+}
+
+// Policy contains fields that are expected to be part of the Policy type
+// within the GraphQL schema.
+type Policy struct {
```

Comment:
why do we have this in cnquery instead of cnspec? can we not move this file over there? feels weird that cnquery defines something policy-related, specially when it's unused

Reply:
Hmmm yea I guess you are right, I just thought since all the previous gql queries where defined in cnquery I should add it here. I'll close this PR and add it to the cnspec one. Good Point